docs/source/guide/zne-3-options.md

```diff
@@ -216,6 +216,66 @@ folding method. Fidelity values must be between zero and one.
 
 +++
 
+#### Folding gates by layer
+
+Circuit debugging techniques (as proposed in
+[arXiv:2204.06056](https://arxiv.org/abs/2204.06056)) utilize the ability to
+apply layerwise folding of a circuit.
+
+A quantum circuit $C$ may be represented as a series of $d$ layers. Each layer
+$L$ contains one or more quantum gates acting concurrently on an $n$-qubit
+system:
+
+$$C = L_d L_{d-1} \ldots L_2 L_1$$.
```

Comment:
Potential stylistic preference, but `\cdots` seems more appropriate here since it's multiplication.

Reply:
Done!